lib/Migration/Version20000Date20240717180417.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * SPDX-FileCopyrightText: 2024 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+namespace OCA\Talk\Migration;
+
+use Closure;
+use OCP\DB\ISchemaWrapper;
+use OCP\Migration\IOutput;
+use OCP\Migration\SimpleMigrationStep;
+
+class Version20000Date20240717180417 extends SimpleMigrationStep {
+	/**
+	 * @param IOutput $output
+	 * @param Closure(): ISchemaWrapper $schemaClosure
+	 * @param array $options
+	 * @return null|ISchemaWrapper
+	 */
+	public function changeSchema(IOutput $output, Closure $schemaClosure, array $options): ?ISchemaWrapper {
+		/** @var ISchemaWrapper $schema */
+		$schema = $schemaClosure();
+
+		// Remove redundant index ta_room from talk_attendees
+		if ($schema->getTable('talk_attendees')->hasIndex('ta_room')) {
+			$schema->getTable('talk_attendees')->dropIndex('ta_room');
```

Comment:
Get the table only once (same for the other case):
```suggestion
		$table = $schema->getTable('talk_attendees');
		if ($table->hasIndex('ta_room')) {
			$table->dropIndex('ta_room');
```